src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -533,19 +533,14 @@ def build_container_from_source_with_buildpack(self, image_name, source, cache_i
                 is_non_supported_platform = False
                 is_non_supported_os = False
                 with subprocess.Popen(command, stdout=subprocess.PIPE) as process:
-
-                    # Collect the standard output in a separate variable that will be printed when a builder
-                    # successfully builds the provided application source
-                    stdout_collection = []
```

Comment:
@cormacpayne FYI I had to remove the delayed logging mechanism because it was making the user experience not great (waiting for the all build to complete to see logs). We should revisit this post-Ignite, but please let me know if there is anything else I should do here.

Reply:
OK sounds good to me